src/main/python/keycodes/keycodes_v5.py

```diff
@@ -576,6 +576,10 @@ class keycodes_v5:
         "RM_SPDU": 0x999b,
         "RM_SPDD": 0x999c,
         "QK_REBOOT": 0x999d,
+
+        "QK_PERSISTENT_DEF_LAYER": 0,
+        "QK_CAPS_WORD_TOGGLE": 0,
+        "QK_LAYER_LOCK": 0,
```

Comment:
potentially my concern is that 0 would reverse resolve to something other than KC_NO, maybe it's better to stick with the invalid keycode numbering scheme?

Reply:
Thanks for pointing that out, I missed that pattern. Revised.